examples/OcclusionSaliency.ipynb

```diff
@@ -87,14 +86,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# The \"application\"\n",
-    "First we will set up our application as a function with defined inputs for\n",
-    "  * the image\n",
-    "  * black-box classifier\n",
-    "  * image perturbation API implementation\n",
-    "  * saliency map generation API implementation\n",
-    "\n",
-    "For the purposes of this tool, let's say that the input blackbox classifier must take in one image (as a numpy matrix) and output a 1D vector of per-class confidences (`Callable[[np.ndarray], np.ndarray]`) for simplicity."
+    "# The \"Application\"\n",
+    "The `xaitk-saliency` package provides a relatively high-level API interface for visual saliency map generation.\n",
+    "This interfaces defines the following input requirements:\n",
+    "  * a reference image\n",
+    "  * a black-box classifier that inferences over images\n",
+    "\n",
+    "As mentioned above, our high-level API accepts black-box classifiers in terms of the `ClassifyImage` interface.\n",
+    "\n",
+    "Further below, we happen to make use of a couple concrete implementations of this interface that happen to perturbation-occlusion sub-pipelines as shown in this diagram.\n",
```

Comment:
I meant to refer to a notebook cell farther down on the page. Maybe it makes more sense to move this diagram down to that section instead of placing it here?